app/grandchallenge/algorithms/templates/algorithms/image_activate.html

```diff
@@ -2,7 +2,7 @@
 {% load crispy_forms_tags %}
 
 {% block title %}
-    Activate Algorithm Image - {{ block.super }}
+    Activate Algorithm Image - {{ object.title }} - {{ block.super }}

```

Comment:
Does this follow the pattern "the object, the thing that the object belongs to, the site" ? Shouldn't this be "Activate Algorithm Image <pk> - Algorithm XYZ - GC?

Reply:
yeah, "the thing that the object belongs to," is the algorithm image since we are activating it, right?
Also, the breadcrumbs is using {{object.title}}, even though it is preceded by "Algorithms".